index.html

```diff
@@ -0,0 +1,159 @@
+      <div class="intro-text">
+        <p>Join 1.000+ attendees in a three-day event and don’t miss the opportunity to meet with the 
+            startups, investors, corporates and developers that are changing the world. Right in the heart of Caracas, 
+            the event will feature 100+ speakers across a variety of panels, keynotes, workshops and fire-side chats on 
+            the current state of blockchain, crypto, DeFi, NFTs, and Web 3.</p>
+      </div>
+      <div class="intro-date">
+        <h2>2025.01.28 (TUE) ~ 2025.01.30 (THU) </h2>
+        <p>@ UCV University Stadium</p>
+      </div>
+    </section>
+
+    <section class="main-program">
+      <div class="main-program-title">
+          <h3>Main Program</h3>
+          <div class="orange-line"></div>
+      </div>
+      <div class="main-program-elements">
+          <div class="main-program-element">
+            <img src="images/main-one.png" alt=""/>
+            <h3>Lecture</h3>
+            <p>Hear from Professionals around the wordl about the cryptocurrincies and how it's gonne be the new money</p>
+          </div>
+          <div class="main-program-element mid-program">
+            <img src="images/main-two.png" alt=""/>
+            <h3>Cryptos Exhibition</h3>
+            <p>
+                Presentation of the best crypto projects and new interesting projects for this year</p>
+          </div>
+          <div class="main-program-element mid-program">
+            <img src="images/main-three.png" alt=""/>
+            <h3>Forum</h3>
+            <p>Have the time to share your thoughts and opinions with experts in each topics</p>
+          </div>
+          <div class="main-program-element mid-program">
+            <img src="images/main-four.png" alt=""/>
+            <h3>Trade</h3>
+            <p>Learn the basics to the advanced to be able to trade and obtain benefits with cryptos</p>
+          </div>
+          <div class="main-program-element mid-program">
+            <img src="images/main-five.png" alt=""/>
+            <h3>Crypto Share</h3>
+            <p>Get the opportunity to network with Crypto New Order affliates around the world and get news about the crpto world</p>
+          </div>
+          <div class="join-button">
+              <button type="button">Join to the future</button>
+          </div> 
+      </div>
+    </section>
+
+    <section class="speakers-div">
+      <div class="featured-speakers-title">
+          <h2>Featured Speakers</h2>
+          <div class="orange-line"></div>
+      </div>
+      <div class="featured-speakers"></div>
+      <div class="button-speaker">
+        <button class="more-button" type="button">MORE <span class="more-arrow">v</span></button>
+      </div>
+      <div class="other-speakers"></div>
+    </section>
+
+    <section class="partner-section">
+      <div class="partner-title">
+        <h2>Partner</h2>
+        <div class="orange-line"></div>
+      </div>
+      <div class="partners-logo">
```

Comment:
- [ ] Please make sure to follow the design guidelines for all screens. For the mobile screen the partners section is different from the design. Please reduce the images size and use flex-wrap so the images can align with each other depends on screen width.

### Your design

![image](https://user-images.githubusercontent.com/83803414/163427780-1662b9ff-dd6a-4152-9b28-cc15ef1e8533.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/163427858-0093fc9a-e67e-4e41-8f64-fbda82b60ea6.png)